src/Client/Web/Bit.Client.Web.BlazorUI/Components/Rating/BitRating.razor.cs

```diff
@@ -12,6 +12,8 @@ public partial class BitRating
         [Parameter] public string Icon { get; set; } = "FavoriteStarFill";
         [Parameter] public string UnselectedIcon { get; set; } = "FavoriteStar";
         [Parameter] public RatingSize Size { get; set; }
+        [Parameter] public EventCallback<int> ValueChanged { get; set; }
+        [Parameter] public EventCallback<int> OnChange { get; set; }
 
         [Parameter]
         public bool IsReadonly
```

Comment:
please check if the value is not actually changed the set method returns fast.
just like the BitCheckBox component